src/integrationTest/java/org/opensearch/security/PointInTimeOperationTest.java

```diff
@@ -133,6 +133,23 @@ public static void createTestData() {
         }
     }
 
+    @Before
+    public void cleanUpPits() throws IOException {
+        try (RestHighLevelClient restHighLevelClient = cluster.getRestHighLevelClient(ADMIN_USER)) {
+            GetAllPitNodesResponse existingPitsResponse = restHighLevelClient.getAllPits(DEFAULT);
+            if (!existingPitsResponse.getPitInfos().isEmpty()) {
+                try {
+                    restHighLevelClient.deleteAllPits(DEFAULT);
+                } catch (OpenSearchStatusException ex) {
+                    if (ex.status() != RestStatus.NOT_FOUND) {
```

Comment:
This seems like the API has a bug, delete all shouldn't return not found even if there was no PITs.

Reply:
Ok, I think I see the problem. I believe there should be a return statement after [here](https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/action/search/PitService.java#L73). I will explore further and raise a PR in core to address.

Reply:
Opened a PR against core to fix the underlying issue: https://github.com/opensearch-project/OpenSearch/pull/11711

Reply:
I updated the cleanup method now that core PR is merged.